assets/docs/edrone/models/ulmfit/pl/sp50k_cased/1.md

```diff
@@ -0,0 +1,196 @@
+# Module edrone/ulmfit/pl/sp50k_cased/1
+ULMFiT language model (English, 50k tokens, cased) trained on the Wikipedia corpus.
```

Comment:
Should be this Polish instead of English?

Same for assets/docs/edrone/models/ulmfit/pl/sp50k_uncased/1.md

Reply:
Yes, thanks - it's changed now.